cubfan135/style.css

```diff
@@ -197,6 +181,11 @@ body.dark #preview-area {
   background-color: #222;
 }
 
+body.dark #canvas {
+  box-shadow: 0 0 14.4px 0 #151515, 0 2.2px 3.6px 0 #191919;

```

Comment:
With this shadow:  
![image](https://user-images.githubusercontent.com/44745238/103471642-a4395f80-4da8-11eb-871a-61d5d32fe533.png)

Without this shadow (derives its shadow from under the `#canvas {`:  
![image](https://user-images.githubusercontent.com/44745238/103471645-b0252180-4da8-11eb-84dc-f493b6a638c7.png)

I prefer the latter.

Reply:
@mmaismma Now that I look at it, absolutely agree with the box-shadow. It's been removed. 
Thank you for your feedback, the requested changes have been made :)